extensions/amp-timeago/1.0/timeago.js

```diff
@@ -28,6 +30,8 @@ export function Timeago(props) {
     'locale': locale,
     'cutoff': cutoff,
     'cutoffText': cutoffText,
+    'containSize': containSize = false,
```

Comment:
Shouldn't these all be destructured in the arg? Would you then also add `containSize` to the types file?

Reply:
Yes. Wasn't sure if that other pull request was already merged. Will rebase and add types.

Reply:
Done